SoftLayer/CLI/virt/detail.py

```diff
@@ -156,6 +154,22 @@ def _bw_table(bw_data):
     return table
 
 
+def _get_price_data(billing_item, total_price):
+    """Returns an item table and price rate value in an object"""
+    price_table = formatting.Table(['Item', 'CategoryCode', 'Recurring Price'])
+    price_table.add_row(['Total', '-', total_price])
+    items = billing_item['nextInvoiceChildren']
+    for item in items:
+        if item.get('recurringFee') is not None:
+            total_price += float(item['recurringFee'])
+            price_table.add_row([item['description'], item['categoryCode'], item['recurringFee']])
+    response = {
+        "priceTable": price_table,
+        "totalPrice": total_price,
+    }
+    return response
```

Comment:
I don't think we should be adding these child items to the total_price, and just return the 1 table. The `Total` row should go at the end as well.
```suggestion
    return price_table
```

Reply:
the function was updated to not add item prices in the total_price and only return a table